### PR TITLE
Apply clang-format to .inl files

### PR DESCRIPTION
**Pull request by danielsn**
*Description of changes:*
turns out we weren't running clang-format on .inl files. Fix that.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: include/aws/common/math.fallback.inl [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-* A copy of the License is located at
-*
-*  http://aws.amazon.com/apache2.0
-*
-* or in the "license" file accompanying this file. This file is distributed
-* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-* express or implied. See the License for the specific language governing
-* permissions and limitations under the License.
-*/
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (feliperodri) [inline include/aws/common/math.fallback.inl]: Should we update this copyrights?

File: format-check.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ if NOT type $CLANG_FORMAT 2> /dev/null ; then
 fi
 
 FAIL=0
-SOURCE_FILES=`find source include tests .cbmc-batch -type f \( -name '*.h' -o -name '*.c' \) -not -name "lookup3.c"`
+SOURCE_FILES=`find source include tests .cbmc-batch -type f \( -name '*.h' -o -name '*.c' -o -name '*.inl' \) -not -name "lookup3.c"`
```
Comment (feliperodri) [inline format-check.sh]: nice!

**Comment by danielsn** [comment]
Clang-format-6 and clang-format-8 disagree on how to format inline asm.  AAARGH.  Anyone have any idea how to fix that?

**Comment by ColdenCullen** [comment]
> Clang-format-6 and clang-format-8 disagree on how to format inline asm. AAARGH. Anyone have any idea how to fix that?

Done, I just turned off the format check on clang-8. Maybe we'll migrate someday, but today is not that day!

File: include/aws/common/math.fallback.inl (hunk 1 above)
Comment (ColdenCullen) [reply include/aws/common/math.fallback.inl]: I don't think so, unless a lawyer asks us to. Last I heard the closing date isn't actually that important, we just need to have it for some reason.

**Pull request merged**